ci/plan.yml

```diff
@@ -281,24 +322,28 @@ SUITES:
     workloads:
       imdb_lstm:
         precision: high
-        skip_platforms: [keras-stripe-ocl-neo, keras-stripe-mtl-uhd630]
         perf_threshold: 0.7
-      
+        skip_platforms: [keras-stripe-ocl-neo, keras-stripe-mtl-uhd630, keras-plaid-mtl-uhd630]
       inception_v3:
         precision: high
         platform_overrides:
           keras-plaid-mtl-uhd630:
             prepend_args: [--examples, 256]
           keras-stripe-mtl-uhd630:
-            prepend_args: [--examples, 256]
-          
+            prepend_args: [-examples, 256]
+        skip_platforms: [keras-stripe-mtl-uhd630, keras-stripe-ocl-neo] # whyfail
+
       mobilenet:
         precision: high
+        skip_platforms: [keras-plaid-mtl-uhd630] # whyfail
+        skip_platforms: [keras-plaid-mtl-uhd630, keras-stripe-mtl-uhd630, keras-stripe-ocl-neo] # whyfail
```

Comment:
Multiple `skip_platforms`, not sure what behavior we get with this, what's the intent?

Reply:
just a bad paste.  this plan.yml is going to get pulled soon